source/_integrations/fritzbox.markdown

```diff
@@ -37,6 +39,8 @@ There is currently support for the following device types within Home Assistant:
 - Sensor
 - Switch
 
+Additionally we also support templates.
```

Comment:
I think "templates" in this (Home Assistant) context can be easily mistaken for Home Assistant templates. Can we make this more clear?

Reply:
```suggestion
Additionally, we also support to trigger smarthome templates.
```